x/gov/handler_test.go

```diff
@@ -626,6 +627,23 @@ func TestUpdateElectorate(t *testing.T) {
 			WantCheckErr:   errors.ErrUnauthorized,
 			WantDeliverErr: errors.ErrUnauthorized,
 		},
+		"Update with open proposal should fail": {
+			Msg: UpdateElectorateMsg{
+				ElectorateID: electorateID,
+				Electors:     []Elector{{Address: alice, Weight: 22}},
+			},
+			SignedBy: bobbyCond,
+			ExpModel: &Electorate{
+				Admin:                 bobby,
+				Title:                 "fooo",
+				Electors:              []Elector{{Address: alice, Weight: 22}},
+				TotalWeightElectorate: 22,
+			},
+			Mods: func(ctx weave.Context, proposal *TextProposal) {
```

Comment:
What is this doing???

```
func textProposalFixture(mods ...func(*TextProposal)) TextProposal {
	now := weave.AsUnixTime(time.Now())
	proposal := TextProposal{
		Title:           "My proposal",
		Description:     "My description",
		ElectionRuleID:  weavetest.SequenceID(1),
		ElectorateID:    weavetest.SequenceID(1),
		VotingStartTime: now.Add(-1 * time.Minute),
		VotingEndTime:   now.Add(time.Minute),
		SubmissionTime:  now.Add(-1 * time.Hour),
		Status:          TextProposal_Submitted,
		Result:          TextProposal_Undefined,
		Author:          alice,
	}
	for _, mod := range mods {
		if mod != nil {
			mod(&proposal)
		}
	}
	return proposal
}
```

This status was already the default, not sure why this is different than `All good with update by owner`

Although I must say these test helpers / fixtures are not so easy to follow